src/main/java/com/solvd/webtest/pages/ios/HomePageIos.java

```diff
@@ -0,0 +1,71 @@
+package com.solvd.webtest.pages.ios;
+
+import com.solvd.webtest.pages.base.CategoriesPageBase;
+import com.solvd.webtest.pages.base.HomePageBase;
+import com.solvd.webtest.pages.base.LoginPageBase;
+import com.solvd.webtest.pages.base.SearchResultPageBase;
+import com.zebrunner.carina.utils.factory.DeviceType;
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+import java.time.Duration;
+
+@DeviceType(pageType = DeviceType.Type.IOS_PHONE, parentClass = HomePageBase.class)
+public class HomePageIos extends HomePageBase {
+    @FindBy(css = ".gh-header-item__icon-link.gh-header-item__profile")
+    private ExtendedWebElement myEbayButton;
+
+    @FindBy(id = "kw")
+    private ExtendedWebElement searchField;
+
+    @FindBy(className = "gh-search__submitbtn")
+    private ExtendedWebElement searchButton;
+
+    @FindBy(css = ".gh-header-item__button.gh__needs-js.gh-header-item__menu")
+    private ExtendedWebElement burgerMenu;
+
+    @FindBy(css = ".gh__needs-js.gh-ham-menu__btn[data-gh-tracking*=\"CATEGORIES\"]")
```

Comment:
This button is not a part of HomePage you can access it from other pages, I am pretty sure you can access it from any other page where you can get into menu so it belongs to menu not HomePage. Moreover Andrei asked us to take whole Element not just part of it so you should take li container of button, not just button itself